### ISSUE TITLE
about rebalance loss

### ISSUE THREAD

**Issue by Ru-hulu** [Time0o/pytorch-colorful-colorization#46]
Hi, thank you for your code. I have a question about the class RebalanceLoss in rebalance_loss.py.
My command line is 
--config
D:\GAN_work\colorful-colorization-master\config\vgg.json
--default-config
D:\GAN_work\colorful-colorization-master\config\default.json
--data-dir
D:\GAN_work\colorful-colorization-master\all_image\train
--checkpoint-dir
D:\GAN_work\colorful-colorization-master\checkpoint
--log-file
D:\GAN_work\colorful-colorization-master\log\test_log.txt
--iterations
10
--iterations-till-checkpoint
5

and the variable class_rebal_lambda is 0.5, the forward function of RebalanceLoss will work, however, the backward function of
it is not used when the project is running.
I think it means that the the gradient is not reweight by color.
Could you give me some suggestions?

**Comment by Ru-hulu** [comment]
![image](https://user-images.githubusercontent.com/45810321/155118360-302375a0-d393-4c2b-868a-7148eb680e1c.png)
![image](https://user-images.githubusercontent.com/45810321/155118500-0ee11a9e-9235-4b2c-a2ae-837fb52de266.png)
here are two picture and I hope they will help me to illustrate my question. The program doesn't stop at the point when executing loss.backward()
